src/bika/lims/api/analysisservice.py

```diff
@@ -153,3 +159,38 @@ def get_service_dependencies_for(service):
         "dependencies": dependencies.values(),
         "dependants": dependants.values(),
     }
+
+
+def check_keyword(keyword, instance=None):
+    """Checks if the given service keyword is valid and unique. Returns an
+    error message if not valid. None otherwise
+    """
```

Comment:
Just a general comment:
This function behaves like an AT field validator and (in my opinion) not like an API function. Better always raise an error in API functions or return True/False to notify the caller about the result.